contig_name_converter/converters.py

```diff
@@ -0,0 +1,66 @@
+import requests
+from retry import retry
+
+
+class SequenceAccessionConverter:
+    """Converter that needs to receive the INSDC contig accession that can uniquely identify the sequence to
+    translate the contig name"""
+    contig_alias_url = 'https://wwwdev.ebi.ac.uk/eva/webservices/contig-alias/'
+
+    def __init__(self, target_naming_convention):
+        self.target_naming_convention = target_naming_convention
+        self._cache = {}
+
+    @retry(tries=3)
```

Comment:
suggest adding jitter and backoff parameters as well...

Reply:
Done